x-pack/plugins/triggers_actions_ui/public/application/sections/actions_connectors_list/components/actions_connectors_list.tsx

```diff
@@ -185,23 +184,6 @@ export const ActionsConnectorsList: React.FunctionComponent = () => {
       sortable: false,
       truncateText: true,
     },
-    {
-      field: 'referencedByCount',
-      'data-test-subj': 'connectorsTableCell-referencedByCount',
-      name: i18n.translate(
-        'xpack.triggersActionsUI.sections.actionsConnectorsList.connectorsListTable.columns.referencedByCountTitle',
-        { defaultMessage: 'Actions' }
```

Comment:
@pmuellr Now that I know how the actual intent behind that column (i.e. display # of Alerts), I think we should keep it. I've brought it back under the name `Alerts`. I prefer `Alerts` over `References` because it's more specific. `Reference` could mean "number of actions using this connector" which is what I first thought was happening in that column. 

![image](https://user-images.githubusercontent.com/4016496/80662963-86f90800-8a47-11ea-97be-d002c2aaec2c.png)

Reply:
Ya, that's fine, since Alerts are the only thing that can reference Actions right now - we can revisit this if/when other things start referencing actions ...